### PR TITLE
Publish multiple deployment packages from domain

### PR DESCRIPTION
**Pull request by djorgensendk**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -21,37 +21,64 @@ on:
...
       SOLUTION_FILE_PATH: 'source/GreenEnergyHub.Charges/GreenEnergyHub.Charges.sln'
 
   # Run all tests in 'Tests.dll'
   unit_test:
+    needs: changes
+    if: ${{ needs.changes.outputs.dotnet == 'true' }}
     uses: Energinet-DataHub/.github/.github/workflows/dotnet-solution-ci-test.yml@v8
     with:
-      TESTS_DLL_FILE_PATH: '\tests\source\GreenEnergyHub.Charges\source\GreenEnergyHub.Charges.Tests\bin\Release\net6.0\GreenEnergyHub.Charges.Tests.dll'
+      TESTS_DLL_FILE_PATH: '\source\GreenEnergyHub.Charges\source\GreenEnergyHub.Charges.Tests\bin\Release\net6.0\GreenEnergyHub.Charges.Tests.dll'
```
Comment (dstenroejl) [inline .github/workflows/ci.yml:69]: Are they missing the "code coverage" flag "business" here?

File: .github/workflows/cd.yml [hunk 2]
```diff
@@ -45,4 +54,48 @@ jobs:
...
+      RELEASE_NAME_PREFIX: dotnet
+
+  #
+  # Dispatch deployment request
+  #
+
+  dispatch_deploment_event:
+    runs-on: ubuntu-latest
+    needs: [
+      # DO NOT REMOVE: necessary to create client-payload
```
Comment (dstenroejl) [inline .github/workflows/cd.yml:73]: In wholesale we use the text "Necessary to be able to use its outputs in the dispatch event"

File: .github/workflows/cd.yml [hunk 3]
```diff
@@ -45,4 +54,48 @@ jobs:
...
+      changes,
+      # Infrastructure
+      infrastructure_promote_prerelease,
+      # .NET
+      # dotnet_promote_prerelease,
```
Comment (dstenroejl) [inline .github/workflows/cd.yml:78]: Should be uncommented

**Review by dstenroejl — CHANGES_REQUESTED** [review]
One "must change" (needs) and some inputs

File: .github/workflows/cd.yml (hunk 3 above)
Comment (djorgensendk) [reply .github/workflows/cd.yml]: done

File: .github/workflows/cd.yml (hunk 2 above)
Comment (djorgensendk) [reply .github/workflows/cd.yml]: changed

File: .github/workflows/ci.yml (hunk 1 above)
Comment (djorgensendk) [reply .github/workflows/ci.yml:69]: added

**Review by dstenroejl — APPROVED** [review]

**Pull request merged**